src/Services/ActivationService.cs

```diff
@@ -64,6 +67,11 @@ private async Task HandleActivationAsync(object activationArgs)
         {
             await activationHandler.HandleAsync(activationArgs);
         }
+

```

Comment:
I noticed that the default handler was never used. Added it.
Reference: [microsoft/TemplateStudio/ActivationService.cs](https://github.com/microsoft/TemplateStudio/blob/fd7db0a491217a1a3166cba55af2d0d0d6d4381f/code/TemplateStudioForWinUICs/Templates/_comp/MT/Project/Services/ActivationService.cs#L51-L54)